components/GameCard.js

```diff
@@ -3,13 +3,21 @@ import Link from "next/link";
 import FavouriteButton from "./FavouriteButton";
 import DeleteButton from "./DeleteButton";
 import EditButton from "./EditButton";
+import { useSession } from "next-auth/react";
 
 export default function GameCard({
   game,
   isFavorite,
   toggleFavorite,
   showModal,
 }) {
+  const session = useSession();
+
+  const isLoggedIn =
```

Comment:
you are checking here if the game was added by the user, so maybe rename it to "createdByCurrentUser"